addons/web/models/base_document_layout.py

```diff
@@ -45,6 +45,7 @@ def _default_company_details(self):
         if 'company_name' not in address_format:
             address_format = '%(company_name)s\n' + address_format
             company_data['company_name'] = company_data['company_name'] or company.name
+        address_format += '<br>'
```

Comment:
This seems arbitrary and unwanted if we don't have `l10n_ma`

Reply:
Done